readthedocs/restapi/serializers.py

```diff
@@ -100,6 +100,8 @@ class BuildSerializer(serializers.ModelSerializer):
     """Build serializer for user display, doesn't display internal fields."""
 
     commands = BuildCommandSerializer(many=True, read_only=True)
+    project_slug = serializers.ReadOnlyField(source='project.slug')
+    version_slug = serializers.ReadOnlyField(source='version.slug')
```

Comment:
I needed to add this new fields here since we can't access this inside the template as an object but as the result of the serializer.